data/campaigns/Northern_Rebirth/scenarios/04_Clearing_the_Mines.cfg

```diff
@@ -446,13 +446,13 @@
 
                 [message]
                     speaker=Hamel
-                    message= _ "Bah! They look like fat slugs to me. Come on boys, let’s chop them to pieces before they can leave the canal."
+                    message= _ "Bah! They look like fat slugs to me. Come on, boys, let’s chop them to pieces before they can leave the canal!"
                 [/message]
             [/then]
             [else]
                 [message]
                     speaker=Hamel
-                    message= _ "Ghouls! They have an insatiable hunger for flesh and even one scratch from their claws will weaken you with sickness and disease. Nasty things; let’s chop them to pieces before they can leave the canal."
+                    message= _ "Ghouls! They have an insatiable hunger for flesh. One scratch from their claws will weaken you with sickness and disease. Nasty things; let’s chop them to pieces before they can leave the canal."
```

Comment:
Both feel about the same to me, so I'd prefer "no change" over "change".

Reply:
I thought the extra long sentence hard to follow, so I changed it. If someone else doesn't agree, I'll change it.

Reply:
I agree with the change in this case personally.